runtime/commonTest/src/kotlinx/serialization/features/PolymorphismTest.kt

```diff
@@ -63,4 +65,17 @@ class PolymorphismTest {
         val s = json.stringify(PolymorphicSerializer(PolyDerived::class), obj)
         assertEquals("[kotlinx.serialization.features.PolyDerived,{id:1,s:b}]", s)
     }
+
+    @Test
+    fun testElementDescriptors() {
+        val serializer = PolymorphicSerializer(PolyDerived::class)
+        try {
+            serializer.descriptor.getElementDescriptor(1)
+            fail ("Exception expected")
+        } catch (e: Exception) {
+            if (e !is SerializationException) {
```

Comment:
Looks like you can use `assertFailsWith`

Reply:
Fixed this. Somehow I missed the function when looking for it.